style.css

```diff
@@ -0,0 +1,406 @@
+/*------------------------- Common Styles ----------------------------*/
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,600;0,700;1,300&display=swap');
+
+
+*, *:before, *:after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+:root {
+  --color-background: #ECEEF1;
+  --color-border: #c7c7c7;
+  --color-primary: #0077C5;
+}
+
+body {
+  background-color: var(--color-background);
+  font-size: 1.6rem;
+  font-family: 'Open Sans', sans-serif;
+}
+
+/*------------------------- Header Section ----------------------------*/
+.header-section {
+  display: flex;
+  height: 6rem;
+  align-items: center;
+  justify-content: flex-end;
+  padding: 1rem 3rem;
+}
+
+.header-text {
+  display: flex;
+  flex-direction: column;
+  font-size: 1.4rem;
+  margin-right: 1rem;
+  text-align: right;
+}
+
+.header-text p {
+  color: grey;
+}
+
+.header-text a {
+  text-decoration: none;
+  color: var(--color-primary);
+}
```

Comment:
The code on these lines can be grouped with the codes on the following lines as they do the same thing:

- lines 169-172

- lines  252-254

- lines 272-275